jsk_perception/package.xml

```diff
@@ -92,6 +92,7 @@
   <exec_depend>leveldb</exec_depend>
   <exec_depend>python-fcn-pip</exec_depend>  <!-- pip -->
   <!-- }} install fcn -->
+  <exec_depend version_lt="0.3.7">python-pytesseract-pip</exec_depend>
```

Comment:
@iory Isn't this `0.3.6`? `0.3.7` causes syntax error.

Reply:
OK. Thanks.